components/AppBar/examples/AppBarDelegateForwardingExample.m

```diff
@@ -106,11 +107,8 @@ - (id)init {
 
     self.title = @"Delegate Forwarding";
 
-    UIColor *color = [UIColor colorWithWhite:0.2f alpha:1];
-    _appBar.headerViewController.headerView.backgroundColor = color;
-    MDCAppBarTextColorAccessibilityMutator *mutator =
```

Comment:
Removing the mutator is not in the PR description. Is this a mistake/typo?

Reply:
This is intentional. The color scheme's colors are intended to be chosen such that they are accessible when mapped to components.

We'll have to revisit the accessibility mutators in the future because they're currently making use of to-be-deprecated APIs.